app.js

```diff
@@ -25,11 +25,12 @@ app.get('/', function get(req, res) {
 });
 
 app.get('/responses', function get(req, res) {
-  res.json(smartSurveyAPICall.getData(
+return smartSurveyAPICall.getData(
     config.apiToken,
     config.apiTokenSecret,
-    config.surveyID,
-    smartSurveyAPICall.parseResponse));
+    config.surveyID)
+  .then(results => res.json(results))
+  .catch(error => console.log(error));
```

Comment:
Should you add more error handling than printing to console? Admittedly I don't know how this is going to be used, but would be good for the user to get a meaningful message if Smart Survey is down for instance.